conans/test/integration/toolchains/cmake/test_cmaketoolchain.py

```diff
@@ -497,7 +496,7 @@ def configure(self):
     # Checking the Android variables created
     # Issue: https://github.com/conan-io/conan/issues/11798
     client.run("install . " + settings)
-    conan_toolchain = load(None, os.path.join(client.current_folder, "conan_toolchain.cmake"))
+    conan_toolchain = load(os.path.join(client.current_folder, "conan_toolchain.cmake"))
```

Comment:
```suggestion
    conan_toolchain = client.load("conan_toolchain.cmake")
```
Sorry, my bad, I didn't realize that this was even a simpler case